recommendation/views.py

```diff
@@ -21,54 +23,55 @@ def new_recommendation(request):
         or user_type != UserType.STUDENT
     ):
         return redirect("landingpage:index")
-    if request.method == "POST":
+
+    try:
+        error_count_rec = None
         user = Student.objects.get(username=username)
-        form = RecommendationForm(request.POST)
-        form.user_id = user.pk
-        if form.is_valid():
-            f = form.save(commit=False)
-            f.user_id = user.pk
-            f.save()
-            current_site = get_current_site(request)
-            mail_subject = "Teacher Recommendation."
-            message = render_to_string(
-                "recommendation/sent_recommendation_email.html",
-                {
-                    "user": f,
-                    "domain": current_site.domain,
-                    "uid1": urlsafe_base64_encode(force_bytes(f.pk)),
-                },
-            )
-            to_email = form.cleaned_data["email_address"]
-            email = EmailMessage(mail_subject, message, to=[to_email])
-            email.send()
-            messages.info(
-                request,
-                "An email has been sent to the teacher you added with instructions on how to fill out your recommendation!",  # noqa: E501
+        recommendations = Recommendation.objects.filter(user_id=user.pk)
+        count = recommendations.count()
+
+        if count == RECOMMENDATION_COUNT:
+            error_count_rec = (
+                "The maximum number of recommendations have been requested!"
```

Comment:
Having "You have already requested 2 recommendations, which is the maximum number of recommendations allowed" will help in hiding the other divs (as mentioned above) and still get the message across.